context/context.go

```diff
@@ -58,6 +58,7 @@ type AMFContext struct {
 	HttpIpv4Port                     int
 	HttpIPv4Address                  string
 	HttpIPv6Address                  string
+	ServerIPv4                       string
```

Comment:
I agree it makes more sense to use those terms, especially for the `BindingIPv4`.
To change `HttpIPv4Address` it's just a bit troublesome I guess, since it is used extensively throughout the code, but what we can do is to just change the config parser file, so that it parses `RegisterIPv4` as `HttpIPv4Address`. This way we can easily change it but I'm not sure if it would affect code readability.

Reply:
To illustrate how a configuration file is currently looking on my side, check the snippet below:
```yaml
configuration:
  smfName: SMF
  serverIPv4: MY_POD_IP
  sbi:
    scheme: http
    ipv4Addr: smf-np.free5gc.svc.cluster.local
    port: 29502
```

Reply:
Or there should be some comment here or developer will be confuse as I am. 😂 

For the config, I think we should put serverIPv4 inside sbi scope, because the IP is used for sbi and there are also other server such as ngap needs other IP. So it may be unsuitable to put serverIPv4 outside sbi scope.

Reply:
Then we can go for something like:
```yaml
configuration:
  smfName: SMF
  sbi:
    scheme: http
    registerIPv4: smf-np.free5gc.svc.cluster.local
    bindingIPv4: MY_POD_IP
    port: 29502
```
And then parse the fields as follows:
```
sbi := configuration.Sbi
context.HttpIPv4Address = sbi.RegisterIPv4
context.BindingIPv4 = os.Getenv(sbi.BindingIPv4)
```

What do you think? Should we do it like this then?

Reply:
Looks good.

Additionally, I think we still need to put comment above variable in struct define so that developer can check it and for some IDE can show it out?

Reply:
Sure, the comment would be a nice addition. 
But I didn't get the part with the IDE. What do you mean by "some IDE can show it out"? 😅

Reply:
Some IDEs such as GoLand can show the comment.

![圖片](https://user-images.githubusercontent.com/60502770/86140460-79353100-bb23-11ea-9b92-6160c6781c1c.png)
![圖片](https://user-images.githubusercontent.com/60502770/86140592-9a961d00-bb23-11ea-95d2-53d75e271c83.png)

Reply:
Ok I see. I wasn't aware of this feature. :smile: 

Then should I do the changes and open another PR for each of the NFs?

Reply:
I think you just need to add a new commit and GitHub will auto update this PR.